pkg/proc/mem.go

```diff
@@ -92,6 +96,18 @@ type compositeMemory struct {
 	data    []byte
 }
 
+// CreateCompositeMemory created a new composite memory type using the provided MemoryReadWriter as the
+// underlying memory buffer.
+func CreateCompositeMemory(mem MemoryReadWriter, arch *Arch, regs op.DwarfRegisters, pieces []op.Piece) (*compositeMemory, error) {
+	// This is basically a small wrapper to avoid having to change all callers
+	// of newCompositeMemory since it existed first.
+	cm, err := newCompositeMemory(mem, arch, regs, pieces)
+	if cm != nil {
+		cm.base = FakeAddressBase
```

Comment:
Same as above, it should be fakeAddressUnresolv.

Reply:
Done.